app/src/main/java/com/framgia/youtube_background_01/screen/main/MainActivity.java

```diff
@@ -1,22 +1,24 @@
 package com.framgia.youtube_background_01.screen.main;
 
-import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.design.widget.BottomNavigationView;
+import android.support.design.widget.NavigationView;
 import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
 import android.view.MenuItem;
 import com.framgia.youtube_background_01.R;
 import com.framgia.youtube_background_01.screen.home.HomeFragment;
 
 public class MainActivity extends AppCompatActivity
         implements BottomNavigationView.OnNavigationItemSelectedListener {
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        findViewById(R.id.bottomNavigation).setOnNavigationItemSelectedListener(this);
+        BottomNavigationView navigationView = findViewById(R.id.bottomNavigation);
+        navigationView.setOnNavigationItemSelectedListener(this);
```

Comment:
Cai BottomNavigationView nay no ko cho findViewById(R.id.bottomNavigation).setOnNavigationItemSelectedListener(this)
nhu the nay anh

Reply:
Ok e